src/autocomplete.js

```diff
@@ -101,7 +101,7 @@ export default class Autocomplete {
 
   onResultsMouseDown() {
     this.mouseDown = true
-    this.results.addEventListener('mouseup', () => (this.mouseDown = false), {once: true})
+    setTimeout(() => (this.mouseDown = false), 100)
```

Comment:
Is the 100 ms value important or can it be 0 to indicate it needs to happen sometime soon in the future? Right now, the 100 value is a magic number that we don't know if we should increase or decrease in the future.